src/keys/public.rs

```diff
@@ -54,3 +64,46 @@ impl Serializable<32> for PublicKey {
         Ok(Self(JubJubAffine::from_bytes(bytes)?.into()))
     }
 }
+
+mod tests {
+
+    #[test]
+    fn partial_eq_test() {
+        use super::*;
+        use dusk_jubjub::{JubJubAffine, JubJubScalar};
+        use rand_core::OsRng;
+
+        let sk1 = PublicKey::from(&SecretKey::random(&mut OsRng));
+        let sk2 = PublicKey::from(&SecretKey::random(&mut OsRng));
+
+        assert_eq!(sk1, sk1);
+        assert_ne!(sk1, sk2);
+
+        // With all coordinates being different the points are the same ie.
+        // equalty holds using this technique.
+        let a = JubJubScalar::from(2u64);
+        let b = JubJubScalar::from(7u64);
+        let c = JubJubScalar::from(4u64);
+        let d = JubJubScalar::from(5u64);
+        let e = JubJubScalar::from(567758785u64);
```

Comment:
Either group those variable in a tuple or add `#[allow(clippy::many_single_char_names)]` to the test.

Reply:
Fixed